konig-schemagen/src/main/java/io/konig/schemagen/sql/RdbmsShapeGenerator.java

```diff
@@ -138,21 +137,20 @@ private void process(Shape rdbmsShape, String logicalPath, String prefix, Shape
 						
 					URI newPredicate =  new URIImpl( propertyNameSpace + newLocalName );
 					
-					declarePredicate(newPredicate);
+					declarePredicate(predicate);
```

Comment:
Why did you change this???

We need to declare the newPredicate.  The original predicate should already be declared.